source/reference/telemetry.txt

```diff
@@ -0,0 +1,54 @@
+.. _c2c-telemetry:
+
+=========
+Telemetry
+=========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+.. include:: /includes/opts/telemetry-statement.rst
+
+Data Collection
+---------------
+
+The data ``mongosync`` tracks includes:
+
+- State information like when ``mongosync`` starts, starts committing,
+  or passes through intermediate states
+- Information about the source and destination clusters and the hardware
+  that hosts the ``mongosync`` instance
+- Initialization arguments
+- The number of source and destination writes
+- The estimated number of bytes ``mongosync`` copies
+- The estimated number of events ``mongosync`` applies
+
+``mongosync`` does not track:
```

Comment:
Could we also add that mongosync does not track mongodb connection strings?

Reply:
Done